### PR TITLE
Update user guide for autocomplete and todo

### LINKED ISSUE #154: Week 11: Update UG - Todo and Autocomplete features

### PR DESCRIPTION
**Pull request by Ugholaf**
- Update user guide for autocomplete
- Update user guide for todo

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -264,27 +269,30 @@ Format: `clear`
...
 Adds a todo to your SoConnect.
 
-Format: `todo add d/DESCRIPTION priority/PRIORITY [t/TAG]…​`
-* PRIORITY can only be `low`, `medium`, `high`.
-* The todo list will always be sorted in decreasing priority.
+Format: `todo add d/DESCRIPTION date/DATE pr/PRIORITY [t/TAG]…​`
+
+* `DATE` should be of the format dd-MM-yyyy (e.g. 24-03-2022).
+* `PRIORITY` can only be `low`, `medium`, `high`.
+* The todo list will always be sorted by date from earliest to latest, then follow by decreasing priority.
```
Comment (Eugene-Ong-W-X) [inline docs/UserGuide.md]:
```suggestion
* The todo list will always be sorted by date from earliest to latest (For todos with the same date, they will be sorted in decreasing priority order).
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -309,16 +317,20 @@ Format: `todo clear`
...
+Format: `todo show`, `todo show today`, `todo show date/DATE`, `todo show date/DATE to DATE`, `todo show t/TAG`, `todo show pr/Priority`
 
-Conditions:
-* `all`: Shows all todos.
-* `priority/PRIORITY`: Shows all todos of the specified priority.
+* `todo show`: Shows all todos.
+* `todo show today`: Shows all todos with the date same as the current date.
+* `todo show date/DATE`: show all todos with the specified date.
+* `todo show date/DATE1 to DATE2`: shows all todos with the date from `DATE1` to `DATE2`.
+* `pr/PRIORITY`: Shows all todos with the specified priority.
```
Comment (Eugene-Ong-W-X) [inline docs/UserGuide.md]:
```suggestion
* `todo show pr/PRIORITY`: Shows all todos with the specified priority.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -309,16 +317,20 @@ Format: `todo clear`
...
 * `t/TAG`: Shows all todos with the specified tag.
```
Comment (Eugene-Ong-W-X) [inline docs/UserGuide.md]:
```suggestion
* `todo show t/TAG`: Shows all todos with the specified tag.
```

**Review by Eugene-Ong-W-X — COMMENTED** [review]
Some fixes to note

File: docs/UserGuide.md (hunk 3 above)
Comment (Ugholaf) [reply docs/UserGuide.md]: Okay done!

File: docs/UserGuide.md (hunk 2 above)
Comment (Ugholaf) [reply docs/UserGuide.md]: Done!

**Pull request merged**